versions/3.0.md

```diff
@@ -3019,6 +3024,22 @@ name: api_key
 in: header
 ```
 
+###### JWT Bearer Sample
+
+```json
+{
+  "type": "scheme",
+  "scheme" : "bearer",
+  "bearerFormat" : "JWT",
```

Comment:
The `bearerFormat` may not be necessary, except to indicate that the token will be prefixed by the term `Bearer` when being passed to the client.  More for documentation than anything.